app/javascript/packages/document-capture/components/acuant-capture.scss

```diff
@@ -37,4 +37,12 @@
   .document-capture-file-image--loading {
     @extend %pad-common-id-card;
   }
+  // Styles for the text that appears over the selfie capture screen to help users position their face for a good photo
+  .document-capture-selfie-feedback {
```

Comment:
Following prior comment, an advantage of BEM is it generally avoids issues with needing to qualify (nest) CSS selectors, so I'd expect this to be defined at a top-level.